### ISSUE TITLE
JavaScript and Syntax errors in demo

### ISSUE THREAD

**Issue by peterkuli** [chilipeppr/serial-port-json-server#55]
When I open http://chilipeppr.com/serialport I get some warnings and errors on the console:

[Deprecation] Synchronous XMLHttpRequest on the main thread is deprecated because of its detrimental effects to the end user's experience. For more help, check https://xhr.spec.whatwg.org/.

GET http://chilipeppr.com/js/embed/highlight.pack.js?_=1532512855115 404 (Not Found)

GET http://chilipeppr.com/js/embed/embed.js?_=1532512855116 404 (Not Found)

Will this be fixed?

**Comment by chilipeppr** [comment]
Yeah, this needs updated. That serial port workspace still uses some old widgets that were refactored for the other workspaces and this one didn't get updated. Might be able to tackle this coming weekend.
